Update workflow-syntax-for-github-actions.md (#8120)
github/docs: content/actions/reference/workflow-syntax-for-github-actions.md

For more information, see "[About comparing branches in pull requests](/articles/about-comparing-branches-in-pull-requests)."

## `on.workflow_dispatch.inputs`

> When using `workflow_dispatch` event, you can optionally specify inputs, which will be presented as form elements in the UI. Workflow dispatch inputs are specified with the same format as action inputs. For more information about format see "[Metadata syntax for GitHub Actions](/actions/creating-actions/metadata-syntax-for-github-actions#inputs)."

Reviewer: Since you can execute workflows outside of the UI (e.g. through the GitHub CLI), I would recommend omitting specific reference to the UI:
```suggestion
When using `workflow_dispatch` event, you can optionally specify inputs that are passed to the workflow. Workflow dispatch inputs are specified with the same format as action inputs. For more information about the format see "[Metadata syntax for GitHub Actions](/actions/creating-actions/metadata-syntax-for-github-actions#inputs)."
```

```yaml
on: 
  workflow_dispatch:
    inputs:
      logLevel:
        description: 'Log level'     
        required: true
        default: 'warning'
      tags:
        description: 'Test scenario tags'
        required: false
```

> The triggered workflow receives the inputs in the [`github.event.inputs`](/actions/reference/context-and-expression-syntax-for-github-actions#github-context) context.

Reviewer: To avoid an inline link:
```suggestion
The triggered workflow receives the inputs in the `github.event.inputs` context. For more information, see "[Context and expression syntax for GitHub Actions](/actions/reference/context-and-expression-syntax-for-github-actions#github-context)."
```
